index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" type="text/css" media="screen" href="index.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+      integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <p class="nav-logo">john doe</p>
+        <button class="nav-btn">
+          <i class="fa fa-bars" aria-hidden="true"></i>
+        </button>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="home-section">
+        <h1 class="section-title">Hey There. i'm Alex</h1>
+        <article class="info-container">
+          <h2 class="info-title">I'am a Software Developer</h2>
+          <p class="info">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don't hestiate to contact me.
+          </p>
+        </article>
+        <div class="links-container">
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/git.svg"
+          /></a>
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/linkedin.svg"
+          /></a>
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/angelist.svg"
+          /></a>
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/twitter.svg"
+          /></a>
+          <a href="http://https://github.com/ambrose-kibet" target="_blank"
+            ><img src="./assets/medium.svg"
+          /></a>
+        </div>
+      </section>
+    </main>
+    <img src="./assets/Group.png" alt="decoration" class="decor-img" />
```

Comment:
- [ ] As this image is used only for decoration purposes and not part of the content of the page, it will be semantically more correct to add it as a background using CSS. 🤝️
Kindly remove it from the markup and add it using CSS.
If you want to learn more about this topic, please check out this [discussion].(https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)
 - [ ] One of the objectives of this project is to understand how to parse Figma designs. Try to stick to the template as much as possible.

Your Design
|![image](https://user-images.githubusercontent.com/17243325/204866893-81eefee1-85cb-4856-af0b-50302eb8bbd2.png)
Figma Design
![image](https://user-images.githubusercontent.com/17243325/204867075-2014bbc0-b698-4f99-9761-b9c614c83e2f.png)